ui/app/AppLayouts/Profile/views/DevicesView.qml

```diff
@@ -23,7 +23,7 @@ SettingsContentBase {
 
     Item {
         width: root.contentWidth
-        height: this.childrenRect.height
+        height: parent.height
```

Comment:
I agree that this file is super messed up. I didn't  change to Layouts or anything, because I know it will be refactored soon.
The new design in Figma is completely different from what we have.

Reply:
Ok, in that case I am approving it now.